release-team/role-handbooks/communications/release-blog.md

```diff
@@ -104,7 +104,7 @@ Check out the full details of the Kubernetes 1.XX release in our [release notes]
 
 ### Availability
 
-Kubernetes 1.XX is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.XX.0). To get started with Kubernetes, check out these [interactive tutorials](https://kubernetes.io/docs/tutorials/) or run local Kubernetes clusters using Docker container “nodes” with [kind](https://kind.sigs.k8s.io/). You can also easily install 1.XX using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). 
+Kubernetes 1.XX is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.XX.0). To get started with Kubernetes, check out these [interactive tutorials](https://kubernetes.io/docs/tutorials/) or run local Kubernetes clusters using Docker container “nodes” with [minikube](https://minikube.sigs.k8s.io/). You can also easily install 1.XX using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). 
```

Comment:
```suggestion
Kubernetes 1.XX is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.XX.0). To get started with Kubernetes, check out these [interactive tutorials](https://kubernetes.io/docs/tutorials/) or run local Kubernetes clusters try [minikube start --kuberntes-version=1.XX](https://minikube.sigs.k8s.io/). You can also easily install 1.XX using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). 
```

Reply:
This is having a command also be a link, so I'm going to close this suggestion